samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -263,6 +268,14 @@ public long getChangelogMinCompactionLagMs(String storeName) {
     return getLong(minCompactLagConfigName, getDefaultChangelogMinCompactionLagMs());
   }
 
+  /**
+   * Helper method to get the default RocksDB max manifest file size in bytes for ALL stores, which is default to 1GB.
```

Comment:
s/ALL stores/ANY store

All makes it ambiguous whether it is the sum of all stores or individual size